ST docs -Apache (#4149)
SumoLogic/sumologic-documentation: docs/reuse/apps/opentelemetry/collector-installation.md

import useBaseUrl from '@docusaurus/useBaseUrl';

In this step, we'll install the collector and add a uniquely identifiable tag to these collector.

1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
1. On the **OpenTelemetry Collection** page, click **Add Collector**.

> 1. In the **Set up Collector** step and select platform of the remote host.<br/><img src={useBaseUrl('img/send-data/linux-install.png')} alt="linux-install" style={{border: '1px solid gray'}} width="800"/>

Reviewer: ```suggestion
1. Select platform of the remote host in the **Set up Collector**  section.<br/><img src={useBaseUrl('img/send-data/linux-install.png')} alt="linux-install" style={{border: '1px solid gray'}} width="800"/>
```

1. Enter your **Installation Token**.
1. Under **Tag data on Collector level**, add a new tag, for example “application = Apache” as seen in the screenshot below to identify these collectors as having Apache running on them.

> 1. For **Collector Settings**, leave them as default (unchecked).

Reviewer: ```suggestion
1. For **Collector Settings**, leave it as default.
```

1. Under **Generate and run the command to install the collector**, copy the command and execute it in your system terminal, where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/linux-terminal-installation.png')} alt="Linux-terminal-installation" width="800"/>
1. Wait for the installation process to complete, then click **Next** to proceed.
1. On the next screen, you will see a list of available Source Templates.

:::note

> If you close this Source template creation screen, you can navigate back. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Collection > Source Template**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the Sumo Logic top menu select **Configuration**, and then under **Data Collection** select **Source Template**. You can also click the **Go To...** menu at the top of the screen and select **Source Template**. Kanso-->

Reviewer 2: ```suggestion
If you close this Source template creation screen, you can navigate back <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Collection > Source Template**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/) by selecting **Configuration** > **Data Collection** > **Source Template**. You can also click the **Go To...** menu at the top of the screen and select **Source Template**. Kanso-->
```

Reviewer 3: Sorry, @kimsauce. I can't take that suggestion. It will work for now. But once we remove the Kanso comment tags, it will be messy.
